lib/rucio/api/did.py

```diff
@@ -28,9 +28,11 @@
 from rucio.db.sqla.session import read_session, stream_session, transactional_session
 
 if TYPE_CHECKING:
-    from typing import Any, Dict, Optional
+    from typing import Any, Dict, Optional, List, Iterator
     from sqlalchemy.orm import Session
 
+    DIDListType = List[DIDType]
```

Comment:
I don't think this is correct. `DIDType` is imported above from `from rucio.db.sqla.constants import DIDType` which is just an enum of the actual DID type (File, ...)

Reply:
Changed to List[Dict]